cli/main.cpp

```diff
@@ -92,23 +92,24 @@ int main(int argc, char* argv[])
 #ifdef NDEBUG
     try {
 #endif
-        return exec.check(argc, argv);
+    return exec.check(argc, argv);
 #ifdef NDEBUG
-    } catch (const InternalError& e) {
-        std::cout << e.errorMessage << std::endl;
-    } catch (const std::exception& error) {
-        std::cout << error.what() << std::endl;
-    } catch (...) {
-        std::cout << "Unknown exception" << std::endl;
-    }
-    return EXIT_FAILURE;
+}
```

Comment:
this is unfortunate. astyle handled this piece of code better. but I guess it's a rare case and can accept it.

Reply:
Yea uncrustify can't match up the braces, it throws this error on this file:

```
flag_parens(22): no match for '{' at [93:9] [CallStack:-DEBUG NOT SET-]
```